zulipterminal/ui_tools/views.py

```diff
@@ -1,5 +1,6 @@
 import threading
 import time
+import math
```

Comment:
Isn't `import math` used in your third commit. If so, let's import it only in the commit where it is used.